Quran/QuranURLs.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  URL.swift
+//  Quran
+//
+//  Created by Mohamed Afifi on 5/2/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+
+struct QuranURLs {
```

Comment:
i renamed this from `URL` due to us now having `Foundation.URL` instead of `NSURL`.